samples/nrf9160/lte_ble_gateway/README.rst

```diff
@@ -37,6 +25,18 @@ The sample also requires a `Nordic Thingy:52`_.
 
 .. include:: /includes/spm.txt
 
+Overview
+*********
+
+The sample connects using Bluetooth LE to a Thingy:52 running the factory pre-loaded application.
```

Comment:
```suggestion
The sample connects using Bluetooth LE to a Thingy:52 running the factory preloaded application.
```
https://projecttools.nordicsemi.no/confluence/display/TECHDOC/Terms%2C+abbreviations%2C+and+spelling+rules#Terms,abbreviations,andspellingrules-P

Reply:
Done.